recipes/ylearn/meta.yaml

```
@@ -0,0 +1,56 @@
+{% set name = "ylearn" %}
+{% set version = "0.1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/DataCanvasIO/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
+  sha256: c54fa3e18ad6dce9e7ac0997db809572e7ebe2cf22be502aabcc84c3282786f6
+
+build:
+  number: 0
```

You can remove the build scripts and call. If the tarball is OK you don't need to inplace build step.

```suggestion
  script: {{ PYTHON }} -m pip install . -vv
  number: 0
```

---

Thank you for your suggestion.
I remove the build scripts and switch tarball source  to pypi.